tests/cli.rs

```diff
@@ -182,7 +181,7 @@ fn register_test() -> Result<(), Box<dyn std::error::Error>> {
         .arg("register")
         .arg("-f=credit_card");
     cmd.assert().success().stdout(predicate::str::contains(
-        "2019-01-01 | 1          | expense-test-acct    | credit_card",
+        "2019-01-01 test memo               expense-test-acct              1           1",
```

Comment:
Definitely not the best way to do this but it works right now to pass the test. It doesn't test the second line but I have not put any thought into how to do the right testing.